### ISSUE TITLE
Infinite loader

### ISSUE THREAD

**Issue by randomdudes** [pentacent/keila#74]
How can I check why my campaign is infinetely loading and is not sending properly?

![image](https://user-images.githubusercontent.com/91569711/135287209-406502a8-65ce-4c80-84df-3d1daed647f1.png)

**Comment by wmnnd** [comment]
Did you end up solving the issue with migrations in #73 ?

**Comment by randomdudes** [comment]
Yes, it can be closed. I used your ops Dockerfiles and it worked better.

**Comment by wmnnd** [comment]
This might indicate that you didn’t set up your mailer correctly. Are you getting any errors in the application log?
